flow-server/src/test/java/com/vaadin/flow/server/frontend/BundleValidationTest.java

```diff
@@ -2121,6 +2152,37 @@ public void defaultProdBundleExists_noCompressedProdBundleFile_noBuildRequired()
                 needsBuild);
     }
 
+    @Test
+    public void defaultProdBundleExists_noCompressedProdBundleFile_reactDisabled_noBuildRequired()
+            throws IOException {
```

Comment:
```suggestion
    public void defaultProdBundleExists_noCompressedProdBundleFile_reactDisabled_buildRequired()
            throws IOException {
```
should be build required. Perhaps `prodBuildRequired`